index.html

```diff
@@ -1,111 +1,173 @@
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+    </section>
+    <section class="our-blog">
+      <h2>Our blog</h2>
+      <article class="sonic-live">
+        <img src="" alt="big-face" class="big-face-image">
+        <div class="p">Tony Nguyen</div>
+        <p class="time">20 Jun 2022</p>
+        <h3>Sonic live</h3>
+        <p>One September 22, 2021 A volutpat nulla eros, nulla ultrices nec nulla imperdiet...</p>
+        <a href="" class="read-more">Read more</a>
+      </article>
+      <a href="" class="read-more">Read more</a>
+    </section>
+    <section class="about-us">
+       <h2>About us</h2>
+       <div class="about-us-text">
+        <p class="text">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla partiatur. Excepteur sint occae cat cupidatat non proident, sunt in culpa qui officia dese runt mollit anim id est labourm velit esse cillum odlore eu fugiat nulla pariatu epreur sint occaecat</p>
+       </div>
+      <div class="high-quality">
+        <img src="" alt="" class="number01">
+        <h3>High quality</h3>
+        <p>Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="big-collection">
+        <img src="" alt="" class="number02">
+        <h3>Big collection</h3>
+        <p>Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="top-resourse">
+        <img src="" alt="" class="number03">
+        <h3>Top resourse</h3>
+        <p>Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <div class="Big-community">
+        <img src="" alt="" class="number04">
+        <h3>Big community</h3>
+        <p>Lorem Ipsum in simply dummy text of the printing and Ipsum typessetting industry.</p>
+       </div>
+       <a href="" class="read-more">Read more</a> 
+    </section>
+    <section class="our-team">
+      <h2>Our team</h2>
+       <div>
+        <img src="" alt="profile-picture" class="profile-picture">
+          <h3>Jason Smith</h3>
+          <p>Co-founder</p>
+        <div class="social-media">
+          <a href="" class="social-media-items"><img src="" alt="Twitter" class="twitter-link"></a>
+          <a href="" class="social-media-items"><img src="" alt="facebook" class="facebook-link"></a>
+          <a href="" class="social-media-items"><img src="" alt="telegram" class="telegram-link"></a>
+          <a href="" class="social-media-items"><img src="" alt="LinkedIn" class="linkedin-link"></a>
+          <a href="" class="social-media-items"><img src="" alt="instargam" class="instagram-link"></a>
         </div>
+       </div>
+     </section>
+    <section class="faq-section">
+      <h2>FAQs</h2>
+        <details class="faq-item">
```

Comment:
@Alinchik1 great usage of details tag! Keep it up!

Reply:
Thank you!